github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-postsubmits.yaml

```diff
@@ -0,0 +1,41 @@
+postsubmits:
+  kubevirt/common-instancetypes:
+    - name: publish-common-instancetypes-builder
+      branches:
+        - main
+      decorate: true
+      max_concurrency: 1
+      extra_refs:
+        - org: kubevirt
+          repo: project-infra
+          base_ref: main
+      labels:
+        preset-podman-in-container-enabled: "true"
+        preset-docker-mirror: "true"
+        preset-gcs-credentials: "true"
+        preset-github-credentials: "true"
+        preset-kubevirtci-quay-credential: "true"
+      annotations:
+        testgrid-create-test-group: "false"
+        rehearsal.allowed: "true"
+      cluster: prow-workloads
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: quay.io/kubevirtci/golang:v20220906-4f60dd3
+            env:
+              - name: COMMON_INSTANCETYPES_CRI
+                value: docker
```

Comment:
this bootstrap image uses podman - while this works probably better to specify podman.